src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -2754,6 +2787,46 @@ protected function castAttribute($key, $value)
 	}
 
 	/**
+	 * Decrypt the given value.
+	 *
+	 * @param  string  $value
+	 * @return mixed
+	 *
+	 * @throws \Exception
+	 */
+	protected function decrypt($value)
+	{
+		if (is_null($value)) return null;
```

Comment:
I'm just following the convention here. All code in the framework returns `null` explicitly.